src/main/java/core/basesyntax/strategy/IncreaseStrategy.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.dto.FruitRecordDto;
+import core.basesyntax.model.Fruit;
+
+public class IncreaseStrategy implements OperationStrategy {
+
+    @Override
+    public int apply(FruitRecordDto fruitRecordDto) {
+        Fruit fruit = fruitRecordDto.getFruit();
+        int updateQuantity = 0;
+        if (!Storage.fruitStorage.containsKey(fruit)) {
+            Storage.fruitStorage.put(fruit, fruitRecordDto.getQuantity());
```

Comment:
return fruitRecordDto.getQuantity() here

Reply:
As you add new value to db

Reply:
done